src/Performance.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Performance improvements.
+ */
+class Performance {
+
+  /**
+   * WooCommerce scripts to be defer or async loaded.
+   *
+   * @var array
+   */
+  const SCRIPTS_ASYNC_LOAD = [
```

Comment:
Please sort A-Z. 

Why did you decide against the specification of both `async` and `defer` as suggest by @sun in https://github.com/netzstrategen/wordpress-shop-standards/pull/49#discussion_r333558972?

Additionally a good short read: https://flaviocopes.com/javascript-async-defer/#async-and-defer

Reply:
Ok, I assumed `defer` and `async` modes where mutually excluding.